js/w3c/style.js

```diff
@@ -73,15 +73,16 @@ define(
             case "LC-NOTE":
               styleFile += "WD";
               break;
+            case "WG-NOTE":
             case "FPWD-NOTE":
-              styleFile += "WG-NOTE";
+              styleFile += "WG-NOTE.css";
```

Comment:
> On 20 Apr 2016, at 8:31 AM, Shane McCarron <notifications@github.com> wrote:
> 
> In js/w3c/style.js:
> 
> >              case "FPWD-NOTE":
> > -              styleFile += "WG-NOTE";
> > +              styleFile += "WG-NOTE.css";
> Why did you add .css to this? I thought they were using conneg to get the file we want.
> 
It seems the w3c changed their minds about conneg: hence this change. Without the '.css', it no longer returns the style sheet, but a 300 page. 


> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>